adapters/repos/db/migrator_shard_status_ops.go

```diff
@@ -181,18 +181,20 @@ func (m *Migrator) unfreeze(ctx context.Context, idx *Index, class string, unfre
 	eg.SetLimit(_NUMCPU * 2)
 
 	for _, name := range unfreeze {
-		// # is a delineator shall come from RAFT and it's away e.g. tenant1#node1
-		// to identify which node path in the cloud shall we get the data from.
-		// it's made because nodeID could be changed on download based on new candidates
-		// when the tenant is unfrozen
-		split := strings.Split(name, "#")
-		if len(split) < 2 {
-			ec.Add(fmt.Errorf("can't detect the old node name"))
-			continue
-		}
-		name := split[0]
-		nodeName := split[1]
+		name := name
 		eg.Go(func() error {
+			// # is a delineator shall come from RAFT and it's away e.g. tenant1#node1
+			// to identify which node path in the cloud shall we get the data from.
+			// it's made because nodeID could be changed on download based on new candidates
+			// when the tenant is unfrozen
+			split := strings.Split(name, "#")
+			if len(split) < 2 {
+				err := fmt.Errorf("can't detect the old node name")
+				ec.Add(err)
```

Comment:
and we doing something with the errors at the end? I can see only `eg.Wait()` at the end

Reply:
ec is returned in the parent func https://github.com/weaviate/weaviate/blob/90c92eadedbe6daa0fcd79698df5c5879416b014/adapters/repos/db/migrator.go#L396

Reply:
great! thank you